src/components/Buttons/TextUnderlineButton.tsx

```diff
@@ -8,8 +8,10 @@ type Props = {
 
 export const TextUnderlineButton = ({ title, onPress, testID }: Props) => {
   return (
-    <TouchableOpacity testID={testID} onPress={onPress}>
-      <Text style={styles.textAsButton}>{title}</Text>
+    <TouchableOpacity testID={testID} style={{ alignSelf: "baseline" }}>
```

Comment:
It's exactly what I am doing. If there is more than one property in style then I put it into styles, otherwise it has no sense. It takes the same space in line and then + more code in styles